File: Ruby/1045.rb
a, b, c = gets.chomp.split().map{|x| x.to_f}.sort().reverse()
if ( a >= (b+c) ) then
  puts ("NAO FORMA TRIANGULO")
elsif ( (a ** 2) == ((b ** 2) + (c ** 2)) ) then
  puts ("TRIANGULO RETANGULO")
elsif ( (a ** 2) > ((b ** 2) + (c ** 2)) ) then
  puts ("TRIANGULO OBTUSANGULO")
elsif ( (a ** 2) < ((b ** 2) + (c ** 2)) ) then
  puts ("TRIANGULO ACUTANGULO")
end

if ( a == b and a == c) then
  puts ("TRIANGULO EQUILATERO")
elsif ( a == b or a == c or b == c) then
  puts ("TRIANGULO ISOSCELES")
end

File: Python/1045.py
entrada = list(map(float, input().split()))
entrada.sort(reverse=True)
a = entrada[0]
b = entrada[1]
c = entrada[2]
if (a >= (b+c)):
    print("NAO FORMA TRIANGULO")
elif ( (a ** 2) == ((b ** 2) + (c ** 2)) ):
    print("TRIANGULO RETANGULO")
elif ( (a ** 2) > ((b ** 2) + (c ** 2)) ):
    print("TRIANGULO OBTUSANGULO")
elif ( (a ** 2) < ((b ** 2) + (c ** 2)) ):
    print("TRIANGULO ACUTANGULO")

if ( a == b and a == c):
    print("TRIANGULO EQUILATERO")
elif ( a == b or a == c or b == c):
    print("TRIANGULO ISOSCELES")
